src/appleseed/renderer/modeling/surfaceshader/constantsurfaceshader.cpp

```diff
@@ -120,7 +120,7 @@ namespace
                 &values);
 
             // Initialize the shading result.
-            shading_result.m_main.rgb() = values.m_color.to_rgb(g_std_lighting_conditions);
+            shading_result.m_main.rgb() = values.m_color.reflectance_to_rgb(g_std_lighting_conditions);
```

Comment:
In some cases like this one. I couldn't tell for sure if it was an emission spectrum of a reflective spectrum. I was hoping to get some help, so I wouldn't need to dig the source code too much.

Reply:
I think we should consider this an emission since loosely speaking, it produces "colors" that are directly visible "on screen".